cmd/influxd/server_integration_test.go

```diff
@@ -674,6 +674,12 @@ func runTestsData(t *testing.T, testName string, nodes Cluster, database, retent
 			expected: `{"results":[{"series":[{"name":"cpu","columns":["time","value"],"values":[["2000-01-01T00:00:10Z",30]]}]}]}`,
 		},
 		{
+			name:     "aggregation with no interval",
+			query:    `SELECT count(value) FROM cpu WHERE time = '2000-01-01 00:00:00'`,
+			queryDb:  "%DB%",
+			expected: `{"results":[{"series":[{"name":"cpu","columns":["time","count"],"values":[["2000-01-01T00:00:00Z",2]]}]}]}`,
```

Comment:
Yeah, I've seen test resets that wipe that database, and write all the same data again. Seems kinda pointless. Why not just write a set of data for a given related set of tests, and then execute the various queries. Of course, reset is there when you need it, but if we start needlessly resetting before every write, our test-run times will just grow to many minutes.

As always, it's a judgement call.